### ISSUE TITLE
Better screen reader support in DetailsList

### ISSUE THREAD

**Issue by edglas1** [OfficeDev/office-ui-fabric-react#8749]
Environment Information
OS: Windows 10
Version: 1809(OS Build 18298.1000)
Browser: Google Chrome Version 73.0.3683.103 (Official Build) (64-bit)
Screen Reader: Narrator

Reproduction of the bug:

### User Experience:
As the screen reader reads through a row in the DetailsList, header information nor row number is not read out, so the customer cannot differentiate what data is in which column. In Excel, the screen reader will read out the column header for the cell and the row number. Another approach is to allow navigation between the cells so the screen reader reads them a cell at a time.

### Repro: 

1. Turn on Narrator and open https://uifabric-prod.azurewebsites.net/?example=jhkjhkj#/components/detailslist#Implementation in Edge browser
2. "DetailsList Implementation" page will be displayed
3. Navigate to table elements in scan mode and observe the above behavior.

#### Actual Behavior: 

Column information is read with no pause or announcement of the column header. 

#### Expected Behavior: 
some way for the customer to differentiate which data is in which column.

**Comment by joschect** [comment]
@edglas1 I am not able to reproduce this in edge, it states the column at the end of the text that it's reading about. Could you confirm?

**Comment by edglas1** [comment]
Ok that's interesting. I apologize, I was testing on Chrome not Edge (I updated the description). I confirmed Edge is working (I can step through the columns, and each one reads out the column header). Does that mean it is an issue in Chrome, or should details list do something different such that it works better in Chrome?

**Comment by joschect** [comment]
@edglas1 Chrome and narrator do not work together for a lot of accessibility. NVDA works better with chrome and might get better results. This isn't an issue with details list or chrome. It's mainly that narrator does not work with chrome.

**Comment by edglas1** [comment]
Ok thanks @joschect, I was not aware of that. Do we bugs filed against Chrome?

**Comment by dzearing** [comment]
@aneeshack4 has been helping to coordinate Narrator issues, and could help give some guidance on the process when Narrator doesn't work well in Chrome.

**Comment by aneeshack4** [comment]
@edglas1 the best way to report issues to Narrator is through stack overflow. Since you work at Microsoft, you can search on https://stackoverflow.microsoft.com/ to see if this an existing issue they're aware of/others have asked about and if it's not to create one on the SO and use tag `win-accessibility` and `narrator`. They've been quite responsive in my experience but if they're not, let me know and we can help you get in touch with them directly. 

Externally, this is the process: https://github.com/OfficeDev/office-ui-fabric-react/wiki/Accessibility-Troubleshooting#narrator
